cpu/src/nmi.S

```diff
@@ -82,12 +82,3 @@ _nmi_handler:
 
     mret
 
-.section .init.text, "ax"
```

Comment:
Was this change intentional? The PR description doesn't mention it...

Reply:
Yes. This is unused code. Made it a stand alone commit.